File: app/src/main/java/com/cashzhang/ashley/bean/Token.java
package com.cashzhang.ashley.bean;

public class Token {
    private String access_token;
    private String refresh_token;
    private String plan;
    private String expires_in;
    private String id;

    public String getAccess_token() {
        return access_token;
    }

    public void setAccess_token(String access_token) {
        this.access_token = access_token;
    }

    public String getRefresh_token() {
        return refresh_token;
    }

    public void setRefresh_token(String refresh_token) {
        this.refresh_token = refresh_token;
    }

    public String getPlan() {
        return plan;
    }

    public void setPlan(String plan) {
        this.plan = plan;
    }

    public String getExpires_in() {
        return expires_in;
    }

    public void setExpires_in(String expires_in) {
        this.expires_in = expires_in;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/com/cashzhang/nozdormu/model/Token.kt
package com.cashzhang.nozdormu.model

import com.google.gson.annotations.SerializedName

class Token(

    @SerializedName("access_token")
    var access_token: String,

    @SerializedName("text")
    var refresh_token: String,

    @SerializedName("text")
    var plan: String,

    @SerializedName("text")
    var expires_in: String,

    @SerializedName("text")
    var id: String,
)